NEWS.md

```diff
@@ -27,6 +27,7 @@
 * Setting filters using a list is now deprecated. Use `teal_slices` and `teal_slice` instead.
 * Removed `CDISCFilteredData` and `CDISCFilteredDataset` and implementing `JoinKeys` handling in their parent classes (`FilteredData` and `DefaultFilteredDataset`).
 * Specifying set of filterable columns is done through `include_varnames` and `exclude_varnames` in `teal_slices`. Specifying `attr(, "filterable")` is hard deprecated.
+* Removed private fields `$code` and `$check` from `FilteredData` class and made appropriate changes to constructor and `init_filtered_data`.
```

Comment:
Private fields is not a concern of end user. Please write about changes in a `init_filtered_data` only, that it no o
```suggestion
*  Removed `check` and `code` from formals of `FilteredData$initialize`. Storing arbitrary `code` and `check` in a `FilteredData` class is no longer possible.
```